streams/src/iosX64Test/kotlin/com/FrozenPublisherTests.kt

```diff
@@ -0,0 +1,21 @@
+package com
```

Comment:
Directly in com?

Reply:
Hum, good catch l'll have a look at the test setup